### DISCUSSION TITLE
Automation

### DISCUSSION THREAD

**Discussion by chinagreenelvis** [WinMerge/winmerge#1772] [Q&A]
Is there a way to automate the merging process for two folders? Ideally, what I would like to do is:

1. Select a source
2. Select a destination
3. Copy all unique and different files from source to destination (ignoring reparse points)
4. Delete unique files from destination

all in essentially just a few clicks.

**Comment by sdottaka** [comment]
Unfortunately, WinMerge cannot automate the above operations.
If you uncheck View→Show Identical Items menu item, it will be easier to find unique or different files.
